music/admin.py

```diff
@@ -17,4 +28,19 @@ def add_music(self, request, queryset):
             music.room.add_music(**MusicSerializer(music).data)
         return
 
+    def duplicate_music(self,request,queryset):
+        targetRoom = Room.objects.get(name=request.POST['nrroom'])
+        if targetRoom :
```

Comment:
if Room.objects.get does not find any item, it does not return None : it throws an exception

Just remove the if, django admin will catch the error the way we want.